app-config.yaml

```diff
@@ -49,14 +49,20 @@ integrations:
 proxy:
   '/harness/prod':
     target: 'https://app.harness.io/'
+    allowedHeaders:
```

Comment:
Do we need to push this? This is very specific to harness.

Reply:
If not required for IDP we can remove this.

Reply:
cc: @devesh-harness

Reply:
@Debanitrkl for IDP we need this, but this is not required here, as this application is not running inside the harness app.

Reply:
Done, please check the latest commit